src/main/java/sirius/biz/tenants/TenantUserManager.java

```diff
@@ -662,7 +664,7 @@ public UserInfo findUserByCredentials(@Nullable WebContext webContext, String us
             return result;
         }
 
-        if (acceptApiTokens && checkApiToken(loginData, password)) {
+        if (checkApiToken(loginData, password)) {
```

Comment:
The new version looks good to me. I don't understand, though, why there ever was a `acceptApiTokens` here. Does that setting change meaning with this PR?